tools/cxbe/Main.cpp

```diff
@@ -178,7 +192,8 @@ int main(int argc, char *argv[])
             LogoPtr = &logo;
         }
 
-        Xbe *XbeFile = new Xbe(ExeFile, szXbeTitle, dwTitleId, dwRegions, bRetail, LogoPtr, szDebugPath);
+        Xbe *XbeFile =
+            new Xbe(ExeFile, szXbeTitle, dwTitleId, dwRegions, dwVersion, bRetail, LogoPtr, szDebugPath);
```

Comment:
This formatting change does not belong in this commit and it violates the clang-format rules.